src/platform/plugins/shared/dashboard/public/dashboard_api/save_modal/save_modal.tsx

```diff
@@ -55,9 +72,14 @@ export const DashboardSaveModal: React.FC<DashboardSaveModalProps> = ({
   tags,
   title,
   timeRestore,
+  accessControl,
 }) => {
+  const accessControlClient = useMemo(() => getAccessControlClient(), []);
```

Comment:
Since `getAccessControlClient` returns the same instance, there is no need for `useMemo`. You can just call `getAccessControlClient()` directly as an argument to  `AccessModeContainer`